### PR TITLE
Run JuliaFormatter and add CI

### PR DESCRIPTION
**Pull request by rossviljoen**
This PR makes only formatting changes and updates the formatting CI to match the rest of JuliaGPs

### CODE REVIEW

File: docs/make.jl [hunk 1]
```diff
@@ -3,15 +3,11 @@ using Documenter, BayesianLinearRegressors
...
     repo="https://github.com/willtebbutt/BayesianLinearRegressors.jl/blob/{commit}{path}#L{line}",
     sitename="BayesianLinearRegressors.jl",
     authors="Will Tebbutt <wt0881@my.bristol.ac.uk>",
     assets=String[],
 )
 
-deploydocs(;
-    repo="github.com/willtebbutt/BayesianLinearRegressors.jl",
-)
+deploydocs(; repo="github.com/willtebbutt/BayesianLinearRegressors.jl")
```
Comment (willtebbutt) [inline docs/make.jl:13]:
```suggestion
deploydocs(; repo="github.com/JuliaGaussianProcesses/BayesianLinearRegressors.jl")
```

File: docs/make.jl [hunk 2]
```diff
@@ -3,15 +3,11 @@ using Documenter, BayesianLinearRegressors
 makedocs(;
     modules=[BayesianLinearRegressors],
     format=Documenter.HTML(),
-    pages=[
-        "Home" => "index.md",
-    ],
+    pages=["Home" => "index.md"],
     repo="https://github.com/willtebbutt/BayesianLinearRegressors.jl/blob/{commit}{path}#L{line}",
```
Comment (willtebbutt) [inline docs/make.jl:7]:
```suggestion
    repo="https://github.com/JuliaGaussianProcesses/BayesianLinearRegressors.jl/blob/{commit}{path}#L{line}",
```

File: src/bayesian_linear_regression.jl [hunk 3]
```diff
@@ -35,7 +35,7 @@ AbstractGPs.mean_and_var(fx::FiniteBLR) = (mean(fx), var(fx))
 
 function AbstractGPs.rand(rng::AbstractRNG, fx::FiniteBLR, samples::Int)
     w = fx.f.mw .+ _cholesky(fx.f.Λw).U \ randn(rng, size(fx.x.X, 1), samples)
-    y = fx.x.X' * w .+ _cholesky(fx.Σy).U' * randn(rng, size(fx.x.X, 2), samples)
+    return y = fx.x.X' * w .+ _cholesky(fx.Σy).U' * randn(rng, size(fx.x.X, 2), samples)
```
Comment (willtebbutt) [inline src/bayesian_linear_regression.jl:38]:
```suggestion
    return fx.x.X' * w .+ _cholesky(fx.Σy).U' * randn(rng, size(fx.x.X, 2), samples)
```

**Review by willtebbutt — APPROVED** [review]
Some minor comments, please feel free to merge once addressed. Still not sure what's going on with CI...

**Comment by rossviljoen** [comment]
This is just for formatting, I'll try to fix tests and docs in another PR

**Pull request merged**